dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -671,6 +672,93 @@
 			interrupts = <45 0>;
 			label = "WDOG1";
 		};
+
+		anatop: anatop@400d8000 {
+			compatible = "nxp,imx-anatop";
+			reg = <0x400d8000 0x4000>;
+			label = "CCM_ANA";
+			#clock-cells = <4>;
+			#pll-clock-cells = <4>;
+		};
+
+		iomuxcgpr: iomuxcgpr@400ac000 {
+			compatible = "nxp,imx-pinmux";
+			reg = <0x400AC000 0x4000>;
+			label = "IOMUX_GPR";
+			#pinmux-cells = <2>;
+		};
+
+		sai1: sai@40384000 {
+			compatible = "nxp,mcux-i2s";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			#pinmux-cells = <2>;
+			reg = <0x40384000 0x4000>;
+			clocks = <&ccm IMX_CCM_SAI1_CLK 0x7C 2
+					&ccm IMX_CCM_CS1CDR_PREDIV 0x28 0
+					&ccm IMX_CCM_CS1CDR_PODF 0x28 63>;
```

Comment:
`PREDIV` and `PODF` can be deleted.

Reply:
as this parameter is related to SAI, leave it here for futer use?

Reply:
How about using the clock API `CLOCK_GetClockRootFreq(kCLOCK_Sai1ClkRoot)`. This way we will not need to add these to the device tree as this API should provide the divided clock value